release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+
+{product-title-short} 3.70 ships with a new default policy that allows you to easily identify deployments that are not restricted by any ingress network policy and to trigger violation alerts accordingly.
+
+- The default policy is named *Deployments should have at least one ingress Network Policy*. It is disabled by default.
+
+- This default policy uses a new policy criterion called *Alert on missing ingress Network Policy*.
+
+- To identify pod isolation gaps, you can clone this default policy or create a new one by using the policy criterion and enabling it on selected resources.
+
+[id="spring-vulnerabilities"]
+=== Identifying Spring critical vulnerabilities
+
+A policy to detect Spring Cloud Function RCE vulnerability (CVE-2022-22963) and Spring Framework Spring4Shell RCE vulnerability (CVE-2022-22965) has been added. It is associated with the Critical severity and is enabled by default.
+
+[id="allow-privilege-escalation-rn"]
+=== Improved validation of pod security context
+
+A new policy criterion has been added to validate the value of `allowPrivilegeEscalation` within the Kubernetes security context. You can use this policy criterion to provide alerts when a deployment is configured to allow a container process to gain more privileges than its parent process.
+
+[id="admin-credentials-link-ocp"]
+=== Finding ACS admin credentials easily in the {ocp} console
+
+For customers deploying {product-title-short} to {ocp} using the recommended Operator method, {product-title-short} admin credentials are now directly visible from the {ocp} console. When viewing the Central object, the *Details* tab provides a clickable link to the credentials under *Admin Password Secret Reference*. The displayed credentials are the default generated password or a previously-configured and stored custom secret. See xref:../installing/install-ocp-operator.adoc#verify-central-install-operator_install-ocp-operator[Verifying Central installation] for more information.
+
+[id="scope-increase-rn"]
+=== Increased number of allowed inclusion and exclusion scopes
+Previously, {product-title-short} limited the number of allowed inclusion and exclusion scopes within a scope to ten each. This restriction has been removed.
+
+[id="notable-tech-changes-370"]
+== Notable technical changes
+
+Vulnerability scanning and reporting for {op-system-first} nodes has been disabled until scanning improvements are made for improved accuracy and to support full host-level scanning beyond just Kubernetes components. Currently, {op-system} uses National Vulnerability Database (NVD) vulnerability data for reporting vulnerabilities for Kubernetes components from {op-system}. In the enhanced version, vulnerability reporting will be based on Red Hat published security data. (ROX-10662)
```

Comment:
There's only one change here, but should it have a subheading (for the sake of consistency with a release that has multiple)?